dotcom-rendering/src/components/FetchMoreGalleriesData.importable.tsx

```diff
@@ -0,0 +1,173 @@
+import { css } from '@emotion/react';
+import { isNonNullable, isObject } from '@guardian/libs';
+import { useEffect, useState } from 'react';
+import { decideFormat } from '../lib/articleFormat';
+import { getDataLinkNameCard } from '../lib/getDataLinkName';
+import { addDiscussionIds } from '../lib/useCommentCount';
+import { palette } from '../palette';
+import { type DCRFrontImage } from '../types/front';
+import { type MainMedia } from '../types/mainMedia';
+import type { FETrailType, TrailType } from '../types/trails';
+import { MoreGalleries } from './MoreGalleries';
+import { Placeholder } from './Placeholder';
+
+type Props = {
+	limit: number; // Limit the number of items shown (the api often returns more)
+	discussionApiUrl: string;
+	absoluteServerTimes: boolean;
+	isAdFreeUser: boolean;
+	ajaxUrl: string;
+	guardianBaseUrl: string;
+};
+
+type MoreGalleriesResponse = {
+	heading: string;
+	trails: FETrailType[];
+};
+
+const minHeight = css`
+	min-height: 300px;
+`;
+
+const getMedia = (galleryCount?: number): MainMedia | undefined => {
+	if (typeof galleryCount === 'number') {
+		return { type: 'Gallery', count: galleryCount.toString() };
+	}
+	return undefined;
+};
+
+const toGalleryTrail = (trail: FETrailType, index: number): TrailType => {
+	const format = decideFormat(trail.format);
+	const image: DCRFrontImage | undefined = trail.masterImage
+		? {
+				src: trail.masterImage,
+				altText: '',
```

Comment:
Can we create an issue to look into making alt text available for these images in the data?

Reply:
Nice suggestion. Created ticket in https://github.com/orgs/guardian/projects/88/views/15?sliceBy%5Bvalue%5D=Decommission+AR&pane=issue&itemId=3488300212&issue=guardian%7Cdotcom-rendering%7C14628